cpp/include/cudf/detail/utilities/device_operators.cuh

```diff
@@ -92,7 +92,7 @@ struct DeviceCount {
  * values. Also, this char pointer serves as valid device pointer of identity
  * value for minimum operator on string values.
  */
-__constant__ char max_string_sentinel[5]{"\xF7\xBF\xBF\xBF"};
+static __constant__ char max_string_sentinel[5]{"\xF7\xBF\xBF\xBF"};
```

Comment:
@kaatish for visibility (wrt https://github.com/rapidsai/cudf/pull/7439/files#r583304683 convo).

Reply:
@harrism informs me that he discussed this bit with @karthikeyann, and that they are alright with this part. I have requested Karthik to examine this PR.